app/features/reminder.py

```diff
@@ -0,0 +1,182 @@
+import asyncio
+import datetime
+import zoneinfo
+
+import interactions
+
+
+class Alarm(interactions.Extension):
+    """Alarm / Reminder extension."""
+
+    def __init__(self, bot: interactions.Client) -> None:
+        self.bot = bot
+        print(f"{self.bot.user} loaded alarm extension")
+        self.tz = UserTimezones()
+
+    base = interactions.SlashCommand(name="remindme", description="Alarm base group")
+    set = interactions.SlashCommand(name="set", description="Set base group")
+
+    def _base_in_opts() -> list[interactions.SlashCommandOption]:
```

Comment:
A method inside a class either needs the 'self' parameter, or if it's not using that the method should be marked as a @staticmethod